src/scripts/stock.coffee

```diff
@@ -17,9 +17,9 @@
 
 module.exports = (robot) ->
   robot.respond /stock (?:info|price|quote)?\s?(?:for|me)?\s?@?([A-Za-z0-9.-_]+)\s?(\d+\w+)?/i, (msg) ->
```

Comment:
This could also be changed to not interpret ".-_" as a range from '.' to '_' as it is now, but I felt it out of scope for this fix.